src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -14,7 +14,7 @@
             - first_template ||= template
             %dd
               %span.form-radio
-              - if template.has_icon?
+              - if !User.current.is_nobody? && template.has_icon?
```

Comment:
Uhh I really do not like this. Getting files is possible via the public API, why isn't this?